src/Concordium/Client/Commands.hs

```diff
@@ -1223,7 +1224,8 @@ bakerGenerateKeysCmd =
     "generate-keys"
     (info
       (BakerGenerateKeys <$>
-        optional (strArgument (metavar "FILE" <> help "Target file of generated credentials.")))
+        optional (strArgument (metavar "FILE" <> help "Target file of generated credentials.")) <*>
+        optional (option auto (long "baker-id" <> metavar "BAKERID" <> help "Optionally provide the baker id.")))
```

Comment:
Perhaps it would be a good idea to explain what the meaning of this flag is. What effect it will have.